modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css

```
@@ -97,3 +97,44 @@
 			brightness(89%) contrast(83%);
 	}
 }
+
+.raylife-navbar-button{
+	border: none;
+	border: 1px solid #4C85FF !important;
+	color: #4C85FF !important;
```

Use colors from stylebook

```suggestion
        border: 1px solid var(--color-action-primary-default) !important;
        color: var(--color-action-primary-default) !important;
```